packages/landingpage-new/styles/globals.css

```diff
@@ -1,11 +1,18 @@
 @import '@inovex.de/elements/dist/inovex-elements/inovex-elements.css';
+@import './guide.css';
 
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: Lato, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-  Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+}
+
+:root h1.header-d2,
```

Comment:
What is this for? 🤔

Reply:
I think it's part of an earlier commit and I just auto-formatted the file with prettier